.github/workflows/check-performance-regression.yml

```diff
@@ -48,11 +48,11 @@ jobs:
       - name: Checkout Pull Request Code
         uses: actions/checkout@v5
 
-      - name: Clone JDK 21 Repo
+      - name: Clone JDK 25 Repo
```

Comment:
Since this is getting bumped, we might want to update the `BASELINE_SECONDS` too?